workers/ohsome_quality_analyst/api/api.py

```diff
@@ -309,23 +325,40 @@ async def list_fid_fields():
     return response
 
 
-def remove_item_from_properties(
-    geojson_object: Union[Feature, FeatureCollection],
-    pattern: str,
+def remove_result_item_from_properties(
+    geojson_object: Union[Feature, FeatureCollection], key: str, flatten: bool
 ) -> None:
     """Remove item from the properties of a GeoJSON Feature or FeatureCollection.
 
-    Items matching the given pattern (See 'fnmatch.fnmatch') will be deleted from the
-    properties.
+    If properties are flattened pattern matching (See 'fnmatch.fnmatch') is used to
+    delete the item with given key from the properties, else item with given key will
+    be removed.
     """
 
-    def _remove_item_from_properties(properties: dict, pattern: str) -> None:
+    def _remove_item_from_properties_pattern(properties: dict, pattern: str) -> None:
```

Comment:
Once #309 has been fixed, the way to include/exclude SVG in the response could be reimplemented. For example to not create the SVG in the first place (hand down the parameter to the `create_figure` call).

Reply:
:+1: Please create an issue or PR for that so we don't forget about it after this PR is merged.

Reply:
https://github.com/GIScience/ohsome-quality-analyst/issues/313